libyul/optimiser/StackToMemoryMover.cpp

```diff
@@ -202,5 +308,15 @@ optional<YulString> StackToMemoryMover::VariableMemoryOffsetTracker::operator()(
 		return YulString{util::toCompactHexWithPrefix(m_reservedMemory + 32 * (m_numRequiredSlots - slot - 1))};
```

Comment:
Is there a reason lower slots are assigned the largest memory slot? Wouldn't the opposite be more optimal in terms of memory expansion costs?

Reply:
The main reason is that @chriseth said, the output is easier to read, if the slots are assigned in this order.
Effectively you will end up with slots counting upwards when following control flow from the main entry point this way.
I originally had it the other way round, but it indeed doesn't really matter, so it's mainly aesthetics.

Reply:
Or well, there probably *is* some difference in memory expansion cost in some cases, but I don't think it matters too much and - without thinking about it too much - I guess that cost tends to be lower when using this order, although it doesn't have to be in all cases.